content/legal/_index.md

```diff
@@ -0,0 +1,144 @@
+
+The Website may contain hyperlinks to redirect User to third party websites, including to agency or partner websites, or social networks.
+
+PrestaShop accepts no liability for the websites to which these links lead as it has no control over the various third party websites.
+
+Websites linking to pages on our Website may not reproduce its pages in any way.
+
+**5.  Personal data.**
+
+**5.1. Personal data collected directly on the Website.** We do not collect personal data directly on the Website.**
+
+**5.2. Personal data collected indirectly on the Website.** We collect some of your personal information indirectly through cookies or other tracking devices. **
+
+**What is a cookie?**
+
+Cookies are small text files (usually consisting of letters and numbers) that are stored in the memory of your browser or device when you visit a website or view a message. 
+
+**What cookies do we use on our Website?** 
+
+**Cookie set by Google Analytics ([Privacy of Google Analytics](https://policies.google.com/privacy/partners?hl=en))**:
+
+**_ga**
+- Analytical cookie
+- Allows us to broadcast our advertising messages on the Google network.
+- This cookie has a lifetime of 13 months.
+
+**_gat**
+- Analytical cookie
+- Used to limit the rate of demand.
+- This cookie has a lifetime of 1 minute.
+
+**_gid**
+- Analytical cookie
+- Allows us to broadcast our advertising messages on the Google network.
+- This cookie has a lifetime of 1 day.
+
+**Cookie set by Cloudflare ([Privacy of Cloudflare](https://www.cloudflare.com/privacypolicy/))**:
+
+**__cf_bm**
+- Technical and necessary cookie
+- This cookie is necessary to protect the bots.
+- This cookie has a lifetime of 30 minutes.
+
+**How to manage the deposit and reading of cookies?** 
+
+Some features of our websites, services, applications and tools are made available to you only through the use of these cookies or trackers.
+
+However, refusing to use these technologies may result in the unavailability of certain features of our services.
+ 
+For more information on blocking, deleting or disabling these technologies, please see your browser or device settings.
+
+For third-party cookies, you can configure your browser settings to refuse all third-party cookies.
+
+For Google Chrome, check [this link](https://support.google.com/chrome/answer/95647?hlrm=en).
+For Mozilla Firefox, check [this link](https://support.mozilla.org/en-US/kb/cookies-information-websites-store-on-your-computer).
+For Microsoft Edge, check [this link](https://support.microsoft.com/en-gb/help/4468242/microsoft-edge-browsing-data-and-privacy-microsoft-privacy).
+For Opera, check [this link](https://help.opera.com/en/latest/security-and-privacy/#tracking).
+For Internet Explorer, check [this link](https://support.microsoft.com/en-gb/help/17442/windows-Internet-explorer-delete-manage-cookies#).
+For Apple Safari, check [this link](https://support.apple.com/guide/safari/manage-cookies-and-website-data-sfri11471/mac.).
```

Comment:
This site does not use cookies

Reply:
there are some, coming from google

Reply:
Who could give us some information?

Reply:
Hi @eternoendless and @kpodemski, so legal just confirmed the fact that with GA, we would need to keep this part.